### ISSUE TITLE
Multiple Questions

### ISSUE THREAD

**Issue by snapo** [NVlabs/Jet-Nemotron#3]
1. Will you be willing to release the code to convert a GPQA LLM to MLA (If it dosent use too much compute resources, all models would benefit from this)? 

2. I could not get it out of the Paper you wrote, but how much compute was exactly spent "just" on the conversion from Qwen3 1.7B GPQA -> Jet 2B MLA (without the whole discovery and testing...)? This question is without all the research and knowning now the exact receipt what works well, how long would it take to convert the Base1.7B model to MLA 2B like you did.

3. Complete training time eval time etc. is kinda messed up, can you please clarify it in more detail?
For example here you write you used 32 H100 GPUS
<img width="1337" height="175" alt="Image" src="https://github.com/user-attachments/assets/b7ffb3ef-04b5-48fe-9467-e337f9a655ad" />
But in the Table 12 you write:

<img width="1338" height="432" alt="Image" src="https://github.com/user-attachments/assets/0b5b7713-bec8-4c43-9936-a5a2ea1296e0" />
Here is reported 18'328 hours in total, which would be 573 days on the 32 H100 ... am i missing something? Or was it 32 x 8x H100 because i dont think you waited 1,5 years for the result :-)

4. You created (maybe by accident) a very important graph over the layers, i never did think that specific layers are for specific task.... Would it be possible to to share this graph generation (code) so i can do one graph for LiveCodeBench, ARC-C, AIME25?

<img width="1407" height="453" alt="Image" src="https://github.com/user-attachments/assets/d9a2ecc2-cd00-4138-899f-886cd33e5c45" />

Thank you very much for answers on the questions, I realy hope we can soon move from GPQA to MLA based models..... as speed matters....

**Comment by t1101675** [comment]
Thanks for your questions! 
1. **On GQA to MLA Conversion:** We believe you're referring to GQA (model architecture), not GPQA (dataset). We did not explore the GQA-to-MLA conversion in our paper, so the corresponding code is not available for release. However, we agree this is a promising direction for future research with Jet-Nemotron.
2. **On Qwen Model Conversion:** The specific conversion from Qwen3 1.7B GQA to Jet 2B MLA was not part of our study. For context, our general method for converting a model like Qwen2.5 to the Jet-Nemotron architecture involves a pre-training phase on approximately 400B tokens.
3. **On Training Time Calculation:** To clarify, the metric we reported is **GPU Hours, not GPU Days**. The total compute was 18,328 GPU hours. On our 32-GPU setup, this translates to approximately 573 hours, or 24 days of training time (18,328÷32≈573 hours).
4. **On Code Release:** We are actively working on releasing the code. We appreciate your patience and will provide an update as soon as possible. Stay tuned!
